qa/L0_infer/test.sh

```diff
@@ -72,6 +72,9 @@ export BACKENDS
 ENSEMBLES=${ENSEMBLES:="1"}
 export ENSEMBLES
 
+# Update if the total number of tests changed
+EXPECTED_NUM_TESTS="42"
```

Comment:
Move this up to line 42. Don't need the comment on it... it will become a standard thing in all test.sh that use python unittest